src/routes/nextRoute/nextRoute.ts

```diff
@@ -167,10 +167,7 @@ const updateStoreAfterCreatePassword = (data: DataProps) => {
 
 const getNextScanRoute = (data: DataProps) => {
   const currentOperation = data?.state?.currentOperation;
-  let path;
-  if (currentOperation === OperationType.ADD_CREDENTIAL) {
-    path = TabsRoutePath.CREDENTIALS;
-  }
+  let path = TabsRoutePath.CREDENTIALS;
```

Comment:
It should just be `let path;` - we shouldn't be redirecting to credentials after scanning anymore